frontend/src/app/pages/registration-done/registration-done.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { QuestionService } from '@app/services/question/question.service';
+import { PatientData } from '@app/model/PatientData';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Patient } from '@app/model/Patient';
+import { PatientDataComponent } from '@app/components/patient-data/patient-data.component';
+import { PatientService } from '@app/services/patient/patient.service';
+import { AlertService } from '@app/services/alert/alert.service';
+
+@Component({
+  selector: 'app-registration-done',
+  templateUrl: './registration-done.component.html',
+  styleUrls: ['./registration-done.component.scss']
+})
+export class RegistrationDoneComponent implements OnInit {
+
+  public patientData: Patient;
```

Comment:
zde může být otazník `public patientData?: Patient;`, který bude naznačovat, že `patientData` můžou být `undefined`. Což se může stát, a `<app-patient-data>` komponenta, do které to pak posíláš, s tím počítá, a případně nic neukáže.

Reply:
super to jsem nevedel ze to tu tak funguje :)